backend/inspirehep/disambiguation/tasks.py

```diff
@@ -70,26 +72,97 @@ def disambiguate_signatures(self, clusters):
     db.session.commit()
 
 
+def match_literature_author(author_data, matcher_config):
```

Comment:
```suggestion
def match_literature_author_with_config(author_data, matcher_config):
```
as this one is taking a config...